seacrowd/sea_datasets/wikiann/wikiann.py

```diff
@@ -6,14 +6,13 @@
 
 from seacrowd.utils import schemas
 from seacrowd.utils.configs import SEACrowdConfig
-from seacrowd.utils.constants import (DEFAULT_SEACROWD_VIEW_NAME,
-                                       DEFAULT_SOURCE_VIEW_NAME, Tasks)
+from seacrowd.utils.constants import DEFAULT_SEACROWD_VIEW_NAME, DEFAULT_SOURCE_VIEW_NAME, Tasks, Licenses
```

Comment:
The variables of `DEFAULT_SEACROWD_VIEW_NAME` and `DEFAULT_SOURCE_VIEW_NAME` aren't being referenced anywhere. Consider removing it or reference it by changing hardcoded values of `source` and `seacrowd_seq`

Reply:
adjusted, thank you!